README.md

```diff
@@ -55,7 +55,7 @@
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## About
-nvm is a version manager for [node.js](https://nodejs.org/en/), designed to be installed per-user, and invoked per-shell. `nvm` works on any POSIX-compliant shell (sh, dash, ksh, zsh, bash), in particular on these platforms: unix, macOS, and windows WSL.
+nvm is a version manager for [node.js](https://nodejs.org/en/), designed to be installed per-user, and invoked per-shell. `nvm` works on any POSIX-compliant shell (sh, dash, ksh, zsh, bash), in particular on these platforms: unix, macOS, and windows WSL (but *not* the standard Windows command prompt nor PowerShell).
```

Comment:
See the note under https://github.com/nvm-sh/nvm#important-notes.

Reply:
Yes, I realise it's mentioned in more detail further down, but it's several page-scrolls down and easily missed there. Considering Windows is still in use on 75% of desktops, I think it should be made pretty clear that it's not going to work on the majority of machines without additional prerequisites.

Reply:
It’s not in use on 75% of *developer* desktops, though - i very much doubt it’s anywhere close to half.